docs/tutorials/integrations/kserve-mm/blue-green.md

```diff
@@ -128,153 +124,113 @@ To send inference requests to the model:
-
-```shell
-kubectl get virtualservice wisdom -o yaml
-```
+    In this tutorial, the model source (field `application.veresions[1].storageUri`) for the candidate is the same as the one for the primary version of the model. In a real world example, this would be different. Here, the version label (`app.kubernetes.io/version`) can be used to distinguish between versions.
 
-You can send additional inference requests as described above. They will be handled by both versions of the model.
+When the candidate version is ready, the Iter8 controller will Iter8 will automatically reconfigure the routing so that requests are sent to both versions.
 
-??? note "Sample output"
-    You will see output from both the primary and candidate version of the application, `wisdom-0` and `wisdom-1` respectively.
+### Verify Routing
 
-    `wisdom-0` output:
-    ```
-    app-version: wisdom-0
-    ```
-
-    `wisdom-1` output:
-    ```
-    app-version: wisdom-1
-    ```
+You can verify the routing configuration by inspecting the `VirtualService` and/or by sending requests as described above. Requests will be handled equally by both versions.
 
 ## Modify weights (optional)
 
-You can modify the weight distribution of inference requests as follows:
+To modify the request distribution between versions, add a `weight` to each version. The weights are relative to each other.
 
 ```shell
-cat <<EOF | helm template routing --repo https://iter8-tools.github.io/iter8 routing-actions --version 0.18 -f - | kubectl apply -f -
-appType: kserve-modelmesh
-appName: wisdom
-action: modify-weights
-strategy: blue-green
-appVersions:
-  - weight: 20
-  - weight: 80
+cat <<EOF | helm upgrade --install wisdom $CHARTS/release -f -
+environment: kserve-modelmesh-istio
+application: 
+  metadata:
+    labels:
+      app.kubernetes.io/name: wisdom
+    annotations:
+      serving.kserve.io/secretKey: localMinIO
+  modelFormat: sklearn
+  versions:
+  - metadata:
+      labels:
+        app.kubernetes.io/version: v0
+    storageUri: "s3://modelmesh-example-models/sklearn/mnist-svm.joblib"
+    weight: 30
+  - metadata:
+      labels:
+        app.kubernetes.io/version: v1
+    storageUri: "s3://modelmesh-example-models/sklearn/mnist-svm.joblib"
+    weight: 70
+  strategy: blue-green
 EOF
 ```
 
-Note that using the `modify-weights` action overrides the default traffic split for all future candidate deployments.
-
-As above, you can verify the routing changes.
+Iter8 automatically reconfigures the routing to distribute traffic between the versions based on the new weights.
 
-## Promote candidate
+### Verify Routing
 
-Promoting the candidate involves redefining the primary version of the application and deleting the candidate version.
+You can verify the routing configuration by inspecting the `VirtualService` and/or by sending requests as described above. Seventy percent of requests will now be handled by the candidate version; the remaining thirty percent by the primary version.
```

Comment:
Similar comment as before. From what I recall, numbers under 10 should be spelled out, otherwise, preferably in the numerical form.